src/features/home/courses/lib/getCourseIcon.tsx

```diff
@@ -0,0 +1,18 @@
+import { AngularIcon, AwsLogo, HtmlIcon, JavascriptIcon, NodeJsIcon, ReactIcon } from '@/icons';
+
+const icons = {
+  'JS / Front-end Pre-school RU': <HtmlIcon />,
+  'JS / Front-end EN': <JavascriptIcon />,
+  'JS / Front-end RU': <JavascriptIcon />,
+  React: <ReactIcon />,
+  Angular: <AngularIcon />,
+  'Node.js': <NodeJsIcon />,
+  'AWS Fundamentals': <AwsLogo />,
+  'AWS Cloud Developer': <AwsLogo />,
+};
```

Comment:
If we someday wanted to change course title, we will need to keep in mind that we need to change it in 2 different places (in the future the number could be increased). And this can potentially introduce some unexpected bugs.

I suggest to create constant for each course name and use it in the `courses` object keys, `icons` object and if needed to reuse the constants in other places of the app



```tsx
JS_RU_TITLE = 'JS / Front-end Pre-school RU';

const icons = {
  [JS_RU_TITLE]: <HtmlIcon />,
  // ...
};
```